manifests/zone.pp

```diff
@@ -13,6 +13,7 @@
   $zone_type = 'master',
   $allow_transfer = [],
   $allow_forwarder = [],
+  $allow_update = undef,
```

Comment:
If I do a bool there, I will need a second parameter to hold the list of IPs allowed to update. validate_bool() will fail if an array is passed to it, I think. If the second variable is acceptable, I'll push a commit with that change.